Project/Source/Components/ComponentParticleSystem.h

```diff
@@ -321,6 +325,24 @@ class ComponentParticleSystem : public Component {
 	bool loopAnimation = true;
 	float nCycles = 1.0f;
 
+	// Trail
```

Comment:
Move it after Collision variables or where you want, but keep consistency with all the code